### PR TITLE
Add new data sources to Vantaa parking data import

### PR DESCRIPTION
**Pull request by japauliina**
## Description

Add several new data sources to Vantaa parking data import.

## Context
[Refs](https://trello.com/c/okKjFrjB/1503-vantaan-kadunvarsipys%C3%A4k%C3%B6inti-palvelukarttaan)

## How Has This Been Tested?

Imported the data locally and added an unit test with a mock data.

### CODE REVIEW

File: services/management/commands/update_vantaa_parking_areas.py [hunk 1]
```diff
@@ -57,78 +103,100 @@ def get_multi_geom(self, obj):
...
+            features = parking_areas.query()
+
+            readable_name = data_source["type"].replace("_", " ").strip() + "s"
+            logger.info(f"Found {features.count} {readable_name} for Vantaa")
+            logger.info(f"Importing {readable_name}...")
+
+            updated_parking_areas = []
+            for feature in features:
+                geom = self.get_multi_geom(
+                    GEOSGeometry(str(feature.get("geometry")), srid=SRC_SRID)
```
Comment (mhieta) [inline services/management/commands/update_vantaa_parking_areas.py:168]: Should there be a None-check for `feature.get("geometry")`?

File: services/management/commands/update_vantaa_parking_areas.py [hunk 2]
```diff
@@ -63,6 +76,7 @@
     "Maksullinen": {"sv": "Avgiftsbelagd", "en": "Paid"},
     "Muu": {"sv": "Något annat", "en": "Other"},
     "Varattu päivisin": {"sv": "Bokas dagtid", "en": "Reserved during the day"},
+    "Liityntäpysäköinti": {"sv": "Pendelparkering", "en": "Park & Ride"},
```
Comment (mhieta) [inline services/management/commands/update_vantaa_parking_areas.py:81]: Should these be in `*.po`-translations?

**Review by mhieta — CHANGES_REQUESTED** [review]
Added couple of comments.

File: services/management/commands/update_vantaa_parking_areas.py (hunk 1 above)
Comment (japauliina) [reply services/management/commands/update_vantaa_parking_areas.py:168]: This is part of old code (just moved) but yeah, I could of course add it.

File: services/management/commands/update_vantaa_parking_areas.py (hunk 2 above)
Comment (japauliina) [reply services/management/commands/update_vantaa_parking_areas.py:81]: These are translations for data and not to any literal in the code, so I don't think it is possible?

Comment (mhieta) [reply services/management/commands/update_vantaa_parking_areas.py:81]: Ok, we can skip these then.

File: services/management/commands/update_vantaa_parking_areas.py (hunk 1 above)
Comment (japauliina) [reply services/management/commands/update_vantaa_parking_areas.py]: Added the skip and a test for it.

**Review by mhieta — APPROVED** [review]
LGTM!

**Pull request merged**